core/templates/dev/head/pages/topic_editor/topic_editor.html

```diff
@@ -148,7 +148,6 @@
   <script src="/templates/dev/head/services/StateTopAnswersStatsBackendApiService.js"></script>
   <script src="/templates/dev/head/domain/classifier/AnswerClassificationResultObjectFactory.js"></script>
   <script src="/templates/dev/head/domain/classifier/ClassifierObjectFactory.js"></script>
-  <script src="/templates/dev/head/domain/exploration/ContentIdsToAudioTranslationsObjectFactory.js"></script>
```

Comment:
Are you sure this isn't needed? (Just confirming)

Reply:
@nithusha21 I found this 2 time in this page so removed one! (Found this on line 161)